src/model/transaction/AggregateTransaction.ts

```diff
@@ -400,4 +403,32 @@ export class AggregateTransaction extends Transaction {
     private getInnerTransactionPaddingSize(size: number, alignment: number): number {
         return 0 === size % alignment ? 0 : alignment - (size % alignment);
     }
+
+    /**
+     * @internal
+     * @param receiptHttp ReceiptHttp
+     * @returns {Observable<AggregateTransaction>}
+     */
+    resolveAliases(receiptHttp: ReceiptHttp): Observable<AggregateTransaction> {
+        const transactionInfo = this.checkTransactionHeightAndIndex();
+        return from(this.innerTransactions).pipe(
+            mergeMap((transaction) => transaction.resolveAliases(receiptHttp, transactionInfo.index)),
+            map((transaction) => transaction as InnerTransaction),
+            toArray(),
+        ).pipe(
```

Comment:
Should aggregateTransactionIndex be provided to resolve aliases here? aggregateTransactionIndex seems like it's never provided.

Reply:
outdated